app/containers/wallet/WalletSendPage.js

```diff
@@ -22,14 +20,27 @@ export default class WalletSendPage extends Component<Props> {
     const { intl } = this.context;
     const { uiDialogs } = this.props.stores;
     const { wallets, transactions } = this.props.stores.substores.ada;
-    const { actions } = this.props;
+    const { actions, stores } = this.props;
     const { isValidAddress } = wallets;
     const { calculateTransactionFee, validateAmount, hasAnyPending } = transactions;
     const activeWallet = wallets.active;
 
     // Guard against potential null values
     if (!activeWallet) throw new Error('Active wallet required for WalletSendPage.');
 
+    // Callback that creates a container to avoid the component knowing about actions/stores
+    const dialogCallback = function (props: DialogProps) {
+      return (<WalletSendConfirmationDialogContainer
```

Comment:
I'm not sure if this kind of callback to create a container is really the right idiomatic approach. I tried to search around to see if there's a well-defined way to deal with these cases but couldn't really find anything. It feel like there's a good chance you're not supposed to structure your code so that this ever becomes an issue. If anybody knows more about this, I'd love to hear

Reply:
It seems that this is been constructed right away, but I guess if `WalletSendConfirmationDialogContainer` is not too heavy is completely fine.